assimilation_code/programs/obs_impact_tool/obs_impact_tool.rst

```diff
@@ -136,7 +128,43 @@ namelist.
    | debug           | logical            | If true print out debugging info.                                           |
    +-----------------+--------------------+-----------------------------------------------------------------------------+
 
-| 
+|
+
+Filter Use
+--------
+To use the ``obs_impact_tool`` code during the filter, the following namelist options are required because ``obs_impact_tool`` is used to create the ``control_impact_runtime.txt`` file as specified above.   
```

Comment:
```suggestion
To use the output file from the ``obs_impact_tool`` during the run of filter, the following namelist options are required because ``obs_impact_tool`` is used to create the ``control_impact_runtime.txt`` file as specified above.   
```